activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1629,6 +1629,13 @@ def remove_timestamps_for_alter(table_name, **options)
           remove_columns_for_alter(table_name, :updated_at, :created_at)
         end
 
+        def add_foreign_key_for_alter(from_table, to_table, **options)
```

Comment:
Removing foreign keys has the same problem, could you add support for that here too? The implementation should be very similar.

Reply:
done